pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools"]
+
+[project]
+name = "opt_adapt"
+version = "0.2"
+dependencies = [
+  "pre-commit",
+  "pytest",
+  "ruff",
+]
+authors = [
+  {name = "Joseph G. Wallwork", email = "joe.wallwork@outlook.com"},
+  {name = "Yihan Liu"},
+  {name = "Stephan C. Kramer"},
+]
+maintainers = [
+  {name = "Joseph G. Wallwork", email = "joe.wallwork@outlook.com"},
+]
+description = "PDE-constrained optimisation using mesh adaptive methods"
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+]
+
+[project.urls]
+Homepage = "https://mesh-adaptation.github.io"
+Documentation = "https://mesh-adaptation.github.io/opt_adapt/index.html"
```

Comment:
```suggestion
Documentation = "https://mesh-adaptation.github.io/docs/opt_adapt/index.html"
```

Reply:
Should do similarly with other repos too :)

Reply:
Good spot! I'll actually drop this because I don't plan to host docs for this repo. (Copy-paste error.) The plan is to upstream it into Goalie. But yes, we should do this in the other repos.